modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -13031,11 +13031,11 @@
 
 	&loginModuleResourceConfig;
 
-    <service name="Login Module Stack" discovery="ModuleOptionsDiscoveryComponent" class="ModuleOptionsComponent"
```

Comment:
Why are we removing Login Module Stack? This will break any existing installs.

Reply:
This configuration is broken, I assumed EAP6 did things differently but it seems i was wrong (seems broken to me as well). The obvious choice was to use the other configuration which was more in line with the ModuleOptionsComponent